static/docs/install/completion.md

```diff
@@ -2,6 +2,12 @@
 
 Enjoy working with DVC faster and with fewer typos!
 
+> Note that shell completion is automatically enabled when DVC is installed on
+> MacOS [with Homebrew](/doc/install/macos#install-with-brew), or on
+> Debian/Ubuntu Linux
+> [from deb repository](/doc/install/linux#install-from-repository). Please
```

Comment:
what about rpm?

Reply:
According to #736, these are the only packages where it's included:

> We are now including bash and zsh completion scripts into our deb, rpm and brew packages.

Reply:
so, yes _rpm_ is here :) it's a separate Linux repo for packages, similar to Debian/Ubuntu, but different. 

So, either `Debian/Ubuntu Linux` and `from deb repository` or there should be a second link to the `install from rpm repository`

Reply:
Ah yes sorry I was confused haha, adding... Hold on.

Reply:
OK, added. Please check.